woocommerce/wc-admin
# Make variations and coupons endpoints return zero-value items

Closes #1398, #1668 and #1704.

Follow-up of #1755.

Makes `variations` and `coupons` endpoints return zero-value items.

### Screenshots
_(Notice _Black, US 8.5_ appears in the table even though its values are 0)_
![image](https://user-images.githubusercontent.com/3616980/54027486-fd7eda80-41a1-11e9-9c39-100504644f8b.png)


### Detailed test instructions:
- Go to the _Coupons_ report.
- Try sorting by every column.
- Start a _Comparison_ making sure one of the coupons has all values to 0 (it has not been used during the selected date range).
- Verify the table shows all compared coupons, even the one that has all values to 0.
- Try sorting by every column and verify the 0-value product is correctly sorted.
